### DISCUSSION TITLE
Maxima thinks 1/3 = 3.3333...

### DISCUSSION THREAD

**Discussion by Meep1000** [wxMaxima-developers/wxmaxima#1877] [Q&A] [answered]
I asked Maxima to evaluate:
`bfloat(1/3);`
...and it returned:
`3.333333333333333b-1`
So the result is off of the true value by 3.  I think I have fundamentally failed to understand some simple aspect of Maxima syntax, but I haven't got the faintest idea what that might be.  Could anyone please explain?  I tried ChatGPT, and it seemed convinced that floating point error must be to blame.  But floating point errors I've encountered in other programming languages have been far smaller.

**Comment by daute** [comment, marked as answer]
No. The result is correct.
The result is given in 'scientific notation', the `-1` at the end means: `*10^-1`

And the result is a [normalized number](https://en.wikipedia.org/wiki/Normalized_number) - exactly one digit (<>0) before the comma.

Another example: 10000/3
```
(%i1) bfloat(10000/3);
(%o1                         3.333333333333333b3
```
for example is: 
3333.333 = 3.3333...*10^3 (normalized)

Best regards, Wolfgang

(And this has nothing to do with wxMaxima, which is the graphical frontend for the [Maxima](https://maxima.sourceforge.io/) project, which does the mathematics in the background.)

**Reply by Meep1000** [reply]
Thank you for politely answering my very noobish question.  I really appreciate it.
